site/docs/concepts/web-app.md

```diff
@@ -0,0 +1,272 @@
+* Authenticating with the flowctl CLI.
+
+Some advanced workflows, like granting or revoking permissions and transforming data with **derivations**, aren't available in the web app.
+
+Even if you prefer the command line or plan to perform a task that's only available through flowctl, we recommend you begin your work in the web app;
+it provides a quicker and easier path to create captures and materializations. You can then switch to flowctl to continue working.
+
+## Signing in
+
+You use either a Google or GitHub account to sign into Flow.
+
+![](<./webapp-images/login-screen.png>)
+
+If you've never used Flow before, you'll be prompted to register before being issued a trial account. If you want to use Flow for production workflows or collaborate with team members, you'll need an organizational account.
+[Contact Estuary](mailto:support@estuary.dev) to create a new organizational account or join an existing organization.
+
+## Navigating the web app
+
+When you log into the web app, you land on the **Welcome** page.
+There are four additional pages visible as tabs in the side navigation: **Captures**, **Collections**, **Materializations**, and **Admin**.
+
+The order of the tabs mirrors the order of a basic Data Flow:
+
+import Mermaid from '@theme/Mermaid';
+
+<Mermaid chart={`
+	graph LR;
+		Capture-->Collection;
+        Collection-->Materialization;
+`}/>
+
+While you may choose to [use the tabs in this sequence](../guides/create-dataflow.md), it's not necessary.
+All Flow entities exist individually, outside of the context of complete Data Flow.
+You can use the different pages in the web app to monitor and manage your items in a number of other ways, as described below.
+
+## Captures page
+
+The **Captures** pages shows you a table of existing Flow [captures](./captures.md) to which you have [access](../reference/authentication.md).
+The **New Capture** button is also visible.
+You use the table to monitor your captures.
+
+![](<./webapp-images/capture-page.png>)
+
+**1:** Select all or deselect all.
+
+**2:** Enable, Disable, and Delete buttons. These actions will be applied to the selected table rows. Choose **Disable** to temporarily pause the flow of data, **Enable** to resume, and **Delete** to permanently remove the capture(s).
+
+**3:** Materialize button. When you click this button, you're directed to the **Create materializations** page.
+All the collections of the selected capture(s) will be added to the materialization.
+
+**4:** Filter captures by name.
+
+**5:** Shard status indicator. Shows the status of the task [shard](./advanced/shards.md) that backs this capture.
+
+* **Primary (Green)**: Data is actively flowing through the capture.
+* **Pending (Yellow)**: The capture is attempting to re-connect. Often, you'll see this after you re-enable the capture as Flow backfills historical data.
+* **Failed (Red)**: The capture has failed with an unrecoverable error.
+* **Disabled (White)**: The capture is disabled.
```

Comment:
When identifying disabled and _unknown_ shard status indicators, I would reference their color and fill for both dark and light mode. For example, the shard status indicator for a disabled task is always a circle outline. It is light grey (or white, if that is how it registers to most people) in dark mode and dark blue (or black, if that is how it registers to most people) in light mode. The shard status indicator for an _unknown_ task is always a filled in circle that is light grey (or white) in dark mode and dark blue (or black) in light mode.